src/functions-reference/transform_functions.qmd

```diff
@@ -0,0 +1,537 @@
+\index{{\tt \bfseries cholesky\_factor\_cov\_constrain }!{\tt (vectors y, int M, int N): matrices}|hyperpage}
+
+`matrices` **`cholesky_factor_cov_constrain`**`(vectors y, int M, int N)`<br>\newline
+Takes a free vector `y` and integers `M` and `N` and returns the `M` by `N` Cholesky
+factor of a covariance matrix. Note that `y` must have length `N + choose(N, 2) + (M - N) * N`,
+and `M` must be greater than or equal to `N`.
+{{< since 2.37 >}}
+
+<!-- matrices; cholesky_factor_cov_jacobian; (vectors y, int M, int N); -->
+\index{{\tt \bfseries cholesky\_factor\_cov\_jacobian }!{\tt (vectors y, int M, int N): matrices}|hyperpage}
+
+`matrices` **`cholesky_factor_cov_jacobian`**`(vectors y, int M, int N)`<br>\newline
+Takes a free vector `y` and integers `M` and `N` and returns the `M` by `N` Cholesky
+factor of a covariance matrix. Note that `y` must have length `N + choose(N, 2) + (M - N) * N`,
+and `M` must be greater than or equal to `N`.
+
+This function also increments the `target` variable
+with the corresponding change of variables adjustment.
+{{< since 2.37 >}}
+
+<!-- vectors; cholesky_factor_cov_unconstrain; (matrices x); -->
+\index{{\tt \bfseries cholesky\_factor\_cov\_unconstrain }!{\tt (matrices x): vectors}|hyperpage}
+
+`vectors` **`cholesky_factor_cov_unconstrain`**`(matrices x)`<br>\newline
+Takes a $M$ by $N$ Cholesky factor of a covariance matrix `x` and returns
+the corresponding free vector of length $N + N\choose 2 + (M - N)N$.
+
+{{< since 2.37 >}}
+
+### Correlation matrices
+
+These functions perform the mathematical transform described in the
+[Correlation Matrices section](https://mc-stan.org/docs/reference-manual/transforms.html#correlation-matrix-transform.section).
+
+<!-- matrices; corr_matrix_constrain; (vectors y, int K); -->
+\index{{\tt \bfseries corr\_matrix\_constrain }!{\tt (vectors y, int K): matrices}|hyperpage}
+
+`matrices` **`corr_matrix_constrain`**`(vectors y, int K)`<br>\newline
+Takes a vector `y` and integer `K`, where `length(y) == choose(K, 2)`, and returns
+a `K` by `K` correlation matrix.
+{{< since 2.37 >}}
+
+<!-- matrices; corr_matrix_jacobian; (vectors y, int K); -->
+\index{{\tt \bfseries corr\_matrix\_jacobian }!{\tt (vectors y, int K): matrices}|hyperpage}
+
+`matrices` **`corr_matrix_jacobian`**`(vectors y, int K)`<br>\newline
+Takes a vector `y` and integer `K`, where `length(y) == choose(K, 2)`, and returns
+a `K` by `K` correlation matrix.
+
+This function also increments the `target` variable
+with the corresponding change of variables adjustment.
+{{< since 2.37 >}}
+
+<!-- vectors; corr_matrix_unconstrain; (matrices x); -->
+\index{{\tt \bfseries corr\_matrix\_unconstrain }!{\tt (matrices x): vectors}|hyperpage}
+
+`vectors` **`corr_matrix_unconstrain`**`(matrices x)`<br>\newline
+Takes a $K$ by $K$ correlation matrix `x` and returns the corresponding
+free vector of size $K\choose 2$.
+{{< since 2.37 >}}
+
+### Covariance matrices
```

Comment:
Mention that a covariance matrix is just a positive definite matrix.  We can land in positive semi-definite land due to rounding issues.